src/cdk/v2/destinations/intercom/utils.js

```diff
@@ -289,6 +290,18 @@ const searchContact = async (message, destination, metadata) => {
     lookupField = 'external_id';
     lookupFieldValue = message.traits.external_id;
   }
+  // if lookup field value is not present, we are returning null
+  if (
+    !isDefinedAndNotNull(lookupFieldValue) ||
+    (typeof lookupFieldValue !== 'string' &&
+      typeof lookupFieldValue !== 'number' &&
+      typeof lookupFieldValue !== 'boolean')
+  ) {
+    warn('Lookup field value is not defined or not a string, number, or boolean', {
```

Comment:
Do we need integration name as the prefix or it is automatically added?

Reply:
Yes, we need. I have added that.